### PR TITLE
feat(TDP-4848): loader

### PR DESCRIPTION
**Pull request by frassinier**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4848

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/42/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/42/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/2/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/2/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/6/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/6/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/9/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/9/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/10/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/10/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/13/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/13/)

File: dataprep-webapp/src/app/components/widgets-containers/app-header-bar/app-header-bar-controller.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default class AppHeaderBarCtrl {
 
 	$onInit() {
 		this.viewKey = this.viewKey || 'appheaderbar';
+		console.log(this.viewKey);
```
Comment (ncomont) [inline dataprep-webapp/src/app/components/widgets-containers/app-header-bar/app-header-bar-controller.js]: ;)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/59/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/59/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/61/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/61/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/2/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/2/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/5/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/5/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/18/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/18/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/19/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/19/)

**Pull request merged**